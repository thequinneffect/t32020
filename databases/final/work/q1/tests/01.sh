psql music -c 'select * from Q1 order by album;'
