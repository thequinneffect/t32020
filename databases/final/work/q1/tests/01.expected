     group     |      album      | year 
---------------+-----------------+------
 Grand Forties | Beyond New York | 2019
(1 row)
